### ISSUE TITLE
Any chance to implement GPU?

### ISSUE THREAD

**Issue by martenq1337** [Dobiasd/frugally-deep#141]
Hey, I'm curious about that is there a small chance to implement the GPU support for this great library?

I don't know nothing about this type of developing, or how hard could it be.

**Comment by Dobiasd** [comment]
The last time I wrote some GPU code was probably over 10 years ago when I made a small shader for some graphics effect. :grimacing:

So currently this is quite far away for me, and I can't even estimate how long it would take to learn enough and then to implement it.

Are you interested in learning some GPU programming? :wink:
Ideally would be an implementation which manages to keep the library small instead of introducing tons of large dependencies. :kissing: :notes:
